PROTO_tests/conftest.py

```diff
@@ -19,7 +19,7 @@ def pytest_addoption(parser):
              "Defaults to 2. If Arkouda is not configured for multi_locale, 1 locale is used"
     )
     parser.addoption(
-        "--size", action="store", default="10**8",
+        "--size", action="store", default="10**4",
```

Comment:
what the rationale for lowering the default size here? was this changed just to make testing easier and you forgot to change it back or do you think we should make `10**4` the new default size